test/torch_np/numpy_tests/core/test_getlimits.py

```diff
@@ -127,13 +131,6 @@ def test_finfo_repr(self):
         assert "dtype=float32" in repr_f32
 
 
-@pytest.mark.skip(reason="Instantiate {i,f}info from dtypes.")
-def test_instances():
-    iinfo(10)
-    finfo(3.0)
-
-
-@pytest.mark.skip(reason="MachAr no implemented (does it need to be)?")
```

Comment:
why is this one gone, and why is it not breaking?

Reply:
Not going to break --- `pytest.mark.skip` is just sugar on top of `raise unittest.SkipTest`. 
Will convert to be more uniform, after the CI finishes.

xfails are more painful because there is not exception in unittest, only a decorator. And then nose, pytest, pytorch invent their own ways.
NB: this is the reason there some `@skip  # FIXME: make xfail` here and there. These are too painful to do uniformly, for both python and pytest drivers.

Reply:
Oh, sorry, was off the mark. Why is the skip gone? It was applied to a non-test function. The `test_instances` test is just something we do not support, similar to object arrays etc. (Why on earth you would even write `finfo(42.0)` instead of `finfo(float)`)